internal/pkg/composable/providers/kubernetes/pod.go

```diff
@@ -447,7 +447,7 @@ func generateContainerData(
 							)
 						} else if config.Hints.DefaultContainerLogs {
 							// in case of no package detected in the hints fallback to the generic log collection
-							_, _ = hintsMapping.Put("generic_logs.container_logs.enabled", true)
+							_, _ = hintsMapping.Put("container_logs.enabled", true)
```

Comment:
Changing the key of the mapping is ok but maybe we should be consistent as we have the same key in line 478...

Reply:
Ah good catch 🤦🏽 . @gizas could you also change https://github.com/elastic/beats/pull/35999/files#diff-050ad6dbacd48af0a47953cade64df8f81c2104c232ddc15dfc1feff29d48badR40-R43 accordingly?

Reply:
Done. I also updated the relevant manifests